src/async-syntax-highlighter.js

```diff
@@ -15,41 +17,84 @@ export default (options) => {
       return ReactAsyncHighlighter.loadAstGenerator();
     }
 
-    static registerLanguage = (_, language) => {
-      if (!supportsLanguageRegistering) {
+    static async loadLanguage(language) {
+      const languageLoader = languageLoaders[language];
+      if (typeof languageLoader === 'function') {
+        return languageLoader(ReactAsyncHighlighter.registerLanguage);
+      } else {
+        throw `Language ${language} not supported`
```

Comment:
can we use semicolon here (feel free to add linter/lint rules if that's easier)

Reply:
What would you think about adding prettier?

Reply:
+10000 , can do in a follow up pr once this is merged so it doesn't cloud the diff but anything to automate pedantic style stuff i'm all for, just think it wasn't around when i started the project and i never got around to adding it.

Reply:
I was really struggling in this repo without prettier, 1 year of prettier made me lose my ability to format code hahaha

Reply:
https://github.com/conorhastings/react-syntax-highlighter/issues/148